CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+- Add the `languages` directory to the `.npmignore` whitelist to allow using the language packs, which were previously 
+missing from the npm package.
```

Comment:
hm, a new line? 🤔

Reply:
Yeah, I tried to make the lines not too long, I don't know if this rule should be applied to the changelog file. @wojciechczerniak ?

Reply:
80? Or am I a 🦖 ?